CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Views/Modals/SignInView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  SignInView.swift
+//  SwiftUIPractice1
+//
+//  Created by Summer Crow on 2022-10-13.
+//
+
+import SwiftUI
+
+struct SignInView: View {
```

Comment:
I didn't see these screens on the video

Reply:
Hi Denis and Yura, thanks for the comments, will be looking over them soon. 
A little background on the missing views in the video and the 135 files changed. I initially understood that you wanted us to build an app that manages the courses we take and have a link to the github for each separate course. I didn't realize you wanted all of the courses in one app. So I had 2 xcode projects and on Friday I copied the coures's code into the Course Manager app to make one project and it got pretty buggy. I fixed most on the bugs today and should have something ready very soon. But I had to go back to making 2 separate projects and to access the course app I created an app path address for it from the Course Manager app using something like the code below Are you OK with this approach?

Also moving forward I will be creating new branch weekly with the naming system you mentioned. 

func goToSecondaryApp() {
        //hardcoded values and forced upwrapping to be replaced soon

        let application = UIApplication.shared
        let secondAppPath = "BuildASwiftUIAppforiOS15://"
        guard let websiteURL = URL(string: "https://github.com/sghourab/SwiftUIPractice1") else {
            return
        }
        let appUrl = URL(string: secondAppPath)!
        
        if application.canOpenURL(appUrl) {
            application.open(appUrl)
        } else {
            application.open(websiteURL)
        }
    }

Reply:
@sghourab thanks for the explanation, you did it in an interesting way. Could you send me tomorrow (or when you finish) in Slack the video how it's working. 

Don't create a new branches until we organise your projects

Reply:
Ok sounds good. Its on my study journal and should be on slack now too. Need to compress my future videos.

Reply:
Also, since I've turned it back into 2 projects, I'm sorry there will be a PR this friday with a brand new Project. Also I've left the Courses folder in the CourseManager project, I'm not referring to it anymore. If you like, whatever changes I make in the Course project, I can update those files as well so you can see what I did differently without getting hundreds of changes? Let me know if you prefer that